Many README updates (#84)
Akuli/catris: README.md

python3 -m catris
```

> ## Connecting to the server

Author: ```suggestion
## Finding the server's IP address
```

All players must be in the same network with the computer where the server is running.
For example, the computers could be all connected to the same WiFi network,
or connected to the same router with Ethernet cables.
The server computer itself can also be used for playing the game,
just like any other computer in the network.

Next you need to know the server's IP address within the local network.
When connecting to the server, players need to specify this IP address.
It is **not** same as the network's public IP,
which is what you find by googling "what is my ip address" or similar.

> Instead, you need to ask the operating system what the IP is.

Author: ```suggestion
```

(if you have a mac, please tell me if it works and whether other commands mentioned here work too).
Either way, you will get messy output with the IP address buried somewhere in the middle of it.
For example, on my Linux computer, `ip a` outputs:

```
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host 
       valid_lft forever preferred_lft forever
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group default qlen 1000
    link/ether 10:60:4b:82:57:01 brd ff:ff:ff:ff:ff:ff
    inet 192.168.1.3/24 brd 192.168.1.255 scope global dynamic noprefixroute eth0
       valid_lft 67299sec preferred_lft 67299sec
    inet6 fe80::ab58:f098:59e0:621f/64 scope link noprefixroute 
       valid_lft forever preferred_lft forever
```

Here `192.168.1.3` is my computer's IP address.
It is **NOT** `127.0.0.1`, as that IP address literally says "this computer",
and it won't connect other computers to the server.

> Once you know the server's IP address, players can connect to the server:

Author: ```suggestion

## Connecting to the server

Once you know the server computer's IP address, players can connect to the server:
```

python3 -m catris
```

## Finding the server's IP address

All players must be in the same network with the computer where the server is running.
For example, the computers could be all connected to the same WiFi network,
or connected to the same router with Ethernet cables.
The server computer itself can also be used for playing the game,
just like any other computer in the network.

Next you need to know the server's IP address within the local network.
When connecting to the server, players need to specify this IP address.
It is **not** same as the network's public IP,

> which is what you find by googling "what is my ip address" or similar.

Author: ```suggestion
which is what you find by googling "what is my ip" or similar.
```

If the server is running Windows, you can run `ipconfig` on a command prompt.
On Linux, you can use `ip a`, and I think `ifconfig` works on MacOS
(if you have a mac, please tell me if it works and whether other commands mentioned here work too).
Either way, you will get messy output with the IP address buried somewhere in the middle of it.
For example, on my Linux computer, `ip a` outputs:

```
1: lo: <LOOPBACK,UP,LOWER_UP> mtu 65536 qdisc noqueue state UNKNOWN group default qlen 1000
    link/loopback 00:00:00:00:00:00 brd 00:00:00:00:00:00
    inet 127.0.0.1/8 scope host lo
       valid_lft forever preferred_lft forever
    inet6 ::1/128 scope host 
       valid_lft forever preferred_lft forever
2: eth0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc pfifo_fast state UP group default qlen 1000
    link/ether 10:60:4b:82:57:01 brd ff:ff:ff:ff:ff:ff
    inet 192.168.1.3/24 brd 192.168.1.255 scope global dynamic noprefixroute eth0
       valid_lft 67299sec preferred_lft 67299sec
    inet6 fe80::ab58:f098:59e0:621f/64 scope link noprefixroute 
       valid_lft forever preferred_lft forever
```

Here `192.168.1.3` is my computer's IP address.

> It is **NOT** `127.0.0.1`, as that IP address literally says "this computer",

Author: ```suggestion
It is **NOT** `127.0.0.1`, as that IP address means"this computer",
```
